benchmarks/spack/tursa/gpu/spack.yaml

```diff
@@ -31,8 +57,14 @@ spack:
       externals:
       - spec: automake@1.16.1
         prefix: /usr
+    cmake:
+      externals:
+      - spec: cmake@3.27.4
+        prefix: /home/y07/shared/utils/core/cmake/3.27.4
```

Comment:
I know a path inside `/home` looks suspicious, but this is from the system module!
```console
$ module show cmake
-------------------------------------------------------------------
/home/y07/shared/tursa-modules/cmake/3.27.4:

conflict        cmake
prepend-path    PATH /home/y07/shared/utils/core/cmake/3.27.4/bin
prepend-path    CPATH /home/y07/shared/utils/core/cmake/3.27.4/include
prepend-path    LD_LIBRARY_PATH /home/y07/shared/utils/core/cmake/3.27.4/lib
prepend-path    MANPATH /home/y07/shared/utils/core/cmake/3.27.4/man
-------------------------------------------------------------------
```